pkg/cephfs/nodeserver.go

```diff
@@ -125,30 +128,42 @@ func (ns *NodeServer) NodeStageVolume(ctx context.Context, req *csi.NodeStageVol
 	}
 
 	// It's not, mount now
+	if err = ns.mount(volOptions, req); err != nil {
+		return nil, err
+	}
+
+	glog.Infof("cephfs: successfully mounted volume %s to %s", volID, stagingTargetPath)
+
+	return &csi.NodeStageVolumeResponse{}, nil
+}
+
+func (*NodeServer) mount(volOptions *volumeOptions, req *csi.NodeStageVolumeRequest) error {
```

Comment:
is the refactoring required?

Reply:
yes its required to fix gocyclo static chec